inference-engine/src/plugin_api/cpp_interfaces/impl/ie_infer_request_internal.hpp

```diff
@@ -355,24 +354,51 @@ class InferRequestInternal : virtual public IInferRequestInternal {
      * @brief Checks whether pre-processing step is required for a given input
      * @param info InputInfo corresponding to input blob
      * @param blob Input Blob object corresponding to input info
+     * @param outBlob Blob object in plugin's desired format
      * @return `True` if pre-processing is required, `false` otherwise
      */
-    bool preProcessingRequired(const InputInfo::Ptr& info, const Blob::Ptr& blob) {
+    bool preProcessingRequired(const InputInfo::Ptr& info, const Blob::Ptr& userBlob, const Blob::Ptr& devicveBlob = nullptr) {
```

Comment:
1. `devicveBlob` -> `deviceBlob`
2. Fix name in doxygen comment

Reply:
done